packages/deployer/README.md

```diff
@@ -1,132 +1,61 @@
 
 ## Setup
-
-- Download the [Terraform v0.13.* binary](https://www.terraform.io/downloads.html) and move it to your `PATH`
-- Get an *Access Key ID* and *Secret Access Key* from AWS and insert into `.env` in the package directory (refer to `.env.example`)
-- Install Serverless Framework globally
-
-```sh
-npm install -g serverless
+* Build all the Airnode packages
+```bash
+# Run from the root
+yarn build
+```
+* Make sure `config.json` and `secrets.env` are available in the `config` directory
+```bash
+# From this package (/packages/deployer)
+cp config/config.json.example config/config.json
+cp config/secrets.env.example config/secrets.env
+# Edit both `config.json` and `secrets.env` to reflect your configuration
 ```
 
-## Basics
-
-Airnode is deployed in two steps:
-
-1. Deploy the mnemonic at AWS SSM at the `region` that the serverless functions will be deployed at using Terraform
-
-1. Deploy the serverless functions at the target `region` using Serverless Framework
-
-The mnemonic deployment needs to be performed only when deploying for the first time on a region.
-For the following redeployments, mnemonic deployment must not be repeated.
-
-## Regular user flow
-
-1. Use the `deploy-first-time` for your first deployment.
-Note down the displayed mnemonic.
-Keep the outputted receipt file, or send it to API3 if you are an API3 partner.
-
-1. Fund the master wallet address with the displayed amount to set your Airnode parameters.
-This will be done for you if you are an API3 partner.
-
-1. Use the `redeploy` command when you need to update your node.
-Note that your `airnodeIdShort` from your receipt file must be inserted into the `nodeSettings.airnodeIdShort` field of your `config.json` file.
-This will be done for you if you are an API3 partner.
+## Common user flow
+1. Use the `deploy` command for your first deployment. Write down the displayed mnemonic. Keep the outputted receipt file. Fund the master wallet address with the displayed amount to set your Airnode parameters.
+2. In order to update the Airnode configuration:
+    * Update the `config.json` file
+    * Write the mnemonic into the `secrets.env` file as `MASTER_KEY_MNEMONIC` environment variable
+    * Run the `deploy` command again
+3. Use the `remove` command to remove the Airnode deployment. Use the `-r` option to provide the receipt file.
 
 ## Commands
-
-Note that for all commands, environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_KEY` must exist.
-You can keep them in a `.env` file (refer to `.env.example`) and inject them when needed with
-```sh
-test -f .env && export $(egrep -v '^#' .env | xargs)
+### deploy
+```bash
+Executes Airnode deployments specified in the config file
```

Comment:
```suggestion
Deploys an Airnode instance using the `config.json` and `secrets.env` files. This can be used for a new deployment or to update an existing deployment.
```